src/cli/evaluations/summarize.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import json
+from pathlib import Path
+import click
+import os
+import pandas as pd
+from tsbench.constants import DEFAULT_EVALUATIONS_PATH
+from cli.evaluations._main import evaluations
+
+BASELINES = ["arima", "ets", "prophet", "mqcnn"]
+
+METRICS = ["mase", "smape", "nrmse", "nd", "ncrps"]
+
+DATASETS = ["m3_yearly", "m3_quarterly", "m3_monthly", "m3_other", "m4_quarterly", "m4_monthly", 
```

Comment:
Could we add a "datasets" options so we don't have to modify the file every rime?